Sources/Prometheus/PrometheusCollectorRegistry.swift

```diff
@@ -780,6 +828,22 @@ extension String {
         return true
     }
 
+    fileprivate func isValidHelpText() -> Bool {
```

Comment:
As a security person string sanitization always scares me as you usually get it wrong. Would you have better ideas here?

Reply:
I guess we could `"....".allSatisfy { character in character.isLetter || character.isNumber }` but also allow punctuation etc, or lean into CharacterSet from Foundation...

Reply:
ACK, I'll check tmrw and possibly re-push the last commit and add that in.

Reply:
@ktoso besides adding all overloads, I refactored the `HelpText` sanitization a bit, however still without Foundation in order to limit some side-effects (such as increasing the binary size for adopters). 
Would you be able to review these changes? Much appreciated ❤️